packages/hash/api/src/graphql/resolvers/knowledge/entity/before-update-entity-hooks.ts

```diff
@@ -56,8 +53,9 @@ const userEntityHookCallback: BeforeUpdateEntityHookCallback = async ({
 
   const currentShortname = userModel.getShortname();
 
-  const updatedShortname: string | undefined =
-    updatedProperties[SYSTEM_TYPES.propertyType.shortName.getBaseUri()];
+  const updatedShortname: string | undefined = updatedProperties[
+    SYSTEM_TYPES.propertyType.shortName.getBaseUri()
+  ] as string | undefined;
```

Comment:
Nit: this is equivalent
```suggestion
  const updatedShortname = updatedProperties[
    SYSTEM_TYPES.propertyType.shortName.getBaseUri()
  ] as string | undefined;
```